### ISSUE TITLE
Importer sets representative file

### ISSUE THREAD

**Issue by val99erie** [curationexperts/alexandria#88]
This story depends on #87.  It will depend on what CSV column names are decided in story #87.

**Acceptance Criteria**
- [ ] When I run the importer (after all the background jobs have finished), the correct file should be the representative file 
- [ ] The correct file is the either the shapefile or the geotiff

**Task**
- [ ] Remove the representative file column from the CSV

**Comment by val99erie** [comment]
One possible way to implement this:  

If there are indexed file headers, like `file1_filename`, `file1_mime_type`, `file2_filename`, `file2_mime_type`, then we could have a column called `representative_file` that just says `file2`, or it could have a value that exactly matches the value of `file2_filename`.

**Comment by val99erie** [comment]
This might already be working properly, so there might not be anything more needed.

**Comment by val99erie** [comment]
This story was written back when we thought we might be attaching multiple files in the CSV input file.  It's no longer needed, so I'll close the ticket with no work.
